README.md

```diff
@@ -81,49 +81,75 @@ import { UISizePlugin, ControlsPlugin, Container } from 'springroll-container';
         buttonSlider: '#button-slider-selector', //controls the size of UI buttons
         buttonSize: 0.5, // button size goes from 0.1 to 1.0 (1 point of precision) default = 0.5
       }),
-      //ControlsPlugin also accepts an [optional] initial value for its control sensitivity
-      new ControlsPlugin({
-        sensitivitySlider: '#sensitivity-slider-selector',
-        sensitivity: 0.5, //control sensitivity goes from 0.1 to 1.0 (1 point of precision) default = 0.5
+      //LayersPlugin controls the progressive removal of distracting game layers. I.e. the higher the slider the more layers should be hidden from player view.
+      new LayersPlugin({
+        //Expects an HTML Input Element of type="range"
+        layersSlider: '#layers-slider-selector' // goes from 0.00 to 1.00 (two points of precision)
+      }),
+      //DifficultyPlugin controls the difficulty/speed of the game.
+      new DifficultyPlugin({
+        //Expects an HTML Input Element of type="range"
+        difficultySlider: '#difficulty-slider-selector',
+        difficulty: 0.5 // goes from 0.1 to 1.0 (one points of precision). Default = 0.5
```

Comment:
I'm thinking this field name should be called `default` instead. I imagine we'll also want this for all of the other plugins as well to be consistent/predictable.

Reply:
switched to defaultDifficulty